### PR TITLE
Allows non-JSON inputs for runs

### PR DESCRIPTION
**Pull request by merschformann**
# Description
This update allows the `new_run` method in the Application class to accept text inputs directly, alongside the existing dict and BaseModel inputs.

# Changes
- Added string type as an acceptable input format for runs.
- Modified input size calculation and upload logic to accommodate the new input type.
- Added `upload_to_presigned_url` method to handle upload put requests.

### CODE REVIEW

**Review by larsbeck — APPROVED** [review]

**Review by sebastian-quintero — APPROVED** [review]
LGTM

File: nextmv/cloud/client.py [hunk 1]
```diff
@@ -180,6 +180,39 @@ def request(
 
         return response
 
+    def upload_to_presigned_url(
```
Comment (sebastian-quintero) [inline nextmv/cloud/client.py]: Why do we need this new method? Can't we use the one we already have? What is the main issue tht would prevent us from using it?

On another note, this method doesn't use "self", should it really be a class method?

One drawback of not using the other method is that we have no timeouts, retries, backoff, or anything.

**Review by sebastian-quintero — COMMENTED** [review]
I would like some clarity on why another method is needed! Thanks.

File: nextmv/cloud/client.py (hunk 1 above)
Comment (merschformann) [reply nextmv/cloud/client.py]: We talked about this offline and will aim to keep it with the existing logic as much as possible.

**Pull request merged**